gems/pending/openstack/openstack_handle/handle.rb

```diff
@@ -32,7 +32,14 @@ def self.try_connection(security_protocol)
         # For backwards compatibility take blank security_protocol as SSL
         yield "https", {:ssl_verify_peer => false}
       elsif security_protocol == 'ssl-with-validation'
-        yield "https", {:ssl_verify_peer => true}
+        excon_options = {:ssl_verify_peer => true}
+        ssl_ca_file = VMDB::Config.new("vmdb").config.fetch_path(:ssl, :ssl_ca_file)
+        ssl_ca_path = VMDB::Config.new("vmdb").config.fetch_path(:ssl, :ssl_ca_path)
```

Comment:
Yeah ... this might make things a bit more complicated (since there are likely several callers from the rails app into this lib, so each will have to look up these config values and pass them in).  But, it's definitely better to have them pass it in than to have this code rely on `VMDB`.